monai/networks/nets/swin_unetr.py

```diff
@@ -272,53 +272,50 @@ def __init__(
         self.out = UnetOutBlock(spatial_dims=spatial_dims, in_channels=feature_size, out_channels=out_channels)
 
     def load_from(self, weights):
+        layers1_0: BasicLayer = self.swinViT.layers1[0]  # type: ignore[assignment]
```

Comment:
May I ask the reason for this change here?

Reply:
This was to try and force better typing on the layers coming from the swinViT object. These get typed as tensor or module which isn't correct as it should be BasicLayer. This isn't the perfect fix for the typing issue but it's a little better, we'll revist typing as discussed.

Reply:
Got it, thanks for the explaination!